binder/environment.yml

```diff
@@ -0,0 +1,12 @@
+name: ipyvega-binder
```

Comment:
I love binder support. Can you send this in a separate pull request or is it tied to the serialization?